app/components/ToolPanel/ToolPanel.tsx

```diff
@@ -28,7 +37,39 @@ const ToolPanel = () => {
       </div>
       {/** this will be the mass action toolbar - it should only show when earthquakes are selected */}
       {/* {earthquakes && <EarthquakeActions />} */}
-
+      <div>
+        <h2 className="font-semibold text-lg">Layers</h2>
+        <div className="flex gap-4 py-2">
+          <button
+            onClick={() => {
+              setActiveLayer({
+                daily: !activeLayers.daily,
+                weekly: activeLayers.weekly,
+              });
+            }}
+            className={`flex items-center gap-2 p-2 ${
+              activeLayers.daily ? 'bg-white' : 'bg-gray-200'
+            } rounded-md`}
+          >
+            Daily
+            {activeLayers.daily ? <FaEye /> : <FaEyeSlash />}
+          </button>
+          <button
+            onClick={() => {
+              setActiveLayer({
+                daily: activeLayers.daily,
+                weekly: !activeLayers.weekly,
+              });
+            }}
+            className={`flex items-center gap-2 p-2 ${
+              activeLayers.weekly ? 'bg-white' : 'bg-gray-200'
+            } rounded-md`}
+          >
+            Weekly
+            {activeLayers.weekly ? <FaEye /> : <FaEyeSlash />}
+          </button>
+        </div>
```

Comment:
<img width="296" alt="image" src="https://github.com/user-attachments/assets/931c1457-bd8f-4a00-89f7-39cc184cb0c4">

These buttons are super basic - I haven't fully fleshed out this tool panel and I think it would be cool to have selections for daily/weekly, and the different magnitudes belonging to each. 

TLDR; These will change in appearance